lib/maze.rb

```diff
@@ -7,7 +7,7 @@
 # Glues the various parts of MazeRunner together that need to be accessed globally,
 # providing an alternative to the proliferation of global variables or singletons.
 module Maze
-  VERSION = '8.1.4'
+  VERSION = '8.1.5'
```

Comment:
I think I'd got for 8.2.0 - whilst minor, it does constitute and Enhancement rather than a Fix.